pom.xml

```diff
@@ -35,6 +35,10 @@
 			<artifactId>hamcrest-all</artifactId>
 			<version>1.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+		</dependency>
 		<!--  Java SemVer v0.9.0 (SemVer 2) -->
```

Comment:
Technically this PR equals to `git reset --hard 5783702fba5def89ea42c1b0f3158f49d265c0ed^` and the difference is that we want to keep commit history.